src/org/ods/services/OpenShiftService.groovy

```diff
@@ -155,6 +157,33 @@ class OpenShiftService {
         }
     }
 
+    HelmStatusSimpleData helmStatus(
+        String project,
+        String release
+    ) {
+        def helmStatusData = retrieveHelmStatus(project, release)
+        HelmStatusSimpleData.from(helmStatusData)
+    }
+
+    HelmStatusData retrieveHelmStatus(
+        String project,
+        String release
+    ) {
+        try {
+            def helmStdout = steps.sh(
+                script: "helm -n ${project} status ${release} --show-resources  -o json",
+                label: "Gather Helm status for release ${release} in ${project}",
+                returnStdout: true
+            ).toString().trim()
+            def object = new JsonSlurperClassic().parseText(helmStdout)
+            def helmStatusData = HelmStatusData.fromJsonObject(object)
+            helmStatusData
+        } catch (Exception e) {
```

Comment:
In groovy you can just catch(e), but it's better to catch concrete exceptions, if possible.

Reply:
Surrounding code in OpenShiftService has simply `catch (ex) {`. Not sure this is better, but at least it would be one style.